### ISSUE TITLE
codes

### ISSUE THREAD

**Issue by ADITYA-THEE-CODER** [ADITYA-THEE-CODER/ML-#91]
for i in range(100):
    pred = theta * X
    error = pred - y
    gradient = np.mean(2 * X * error)
    theta = theta - alpha * gradient
    
    if i % 10 == 0:  # Print every 10 iterations
        loss = np.mean(error**2)
        print(f"Iteration {i}: theta={theta:.4f}, loss={loss:.4f}")
